sdk/communication/communication-identity/package.json

```diff
@@ -62,6 +62,9 @@
       "azure",
       "azure-communication-services"
     ],
+    "skip": [
```

Comment:
Why are we skipping this instead if fixing?
Are we planning to  include this tests in test runs later? If so, a TODO comment might be useful

Reply:
yes, we plan to do so:
https://skype.visualstudio.com/SPOOL/_workitems/edit/3105974